### PR TITLE
Create initial structure

### PR DESCRIPTION
**Pull request by lpelizzaMS**
### Background
Create the initial structure and also add a gitignore file.

### Changes

I have followed the structure suggested for Xamarin Forms projects. Then, we have this projects inside the Solution
Doppler.Mobile.iOS (this project has all the files related to iOS platform)
Doppler.Mobile.Android (this project has all the files related to Android platform)
Doppler.Mobile (this project has the logic related to the application like views and view models)
Doppler.Mobile.Core (this project has the logic related to the core of the application like services and models)

### CODE REVIEW

**Comment by andresmoschini** [comment]
I understand that Xamarin Forms has a common UI for any platform. Why did you need to create platform based projects?

**Comment by andresmoschini** [comment]
Hi @MakingSense/doppler-products team, sorry for the spam, but maybe some of you are interested in following the progress of this project, in that case, feel free to watch it and participate in coder reviews and discussions. 

image

File: Doppler.Mobile/Doppler.Mobile/Doppler.Mobile.Android/Doppler.Mobile.Android.csproj [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <WarningLevel>4</WarningLevel>
+    <AndroidManagedSymbols>true</AndroidManagedSymbols>
+    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Mono.Android" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Xml" />
```
Comment (andresmoschini) [inline Doppler.Mobile/Doppler.Mobile/Doppler.Mobile.Android/Doppler.Mobile.Android.csproj]: Is it not possible to use the new VS2017 .csproj  format?

**Comment by lpelizzaMS** [comment]
@andresmoschini 
regarding your question: "Why did you need to create platform-based projects?"

The template of Xamarin Forms has those projects. They are used to configure each platform.

File: Doppler.Mobile/Doppler.Mobile/Doppler.Mobile.Android/Doppler.Mobile.Android.csproj (hunk 1 above)
Comment (andresmoschini) [reply Doppler.Mobile/Doppler.Mobile/Doppler.Mobile.Android/Doppler.Mobile.Android.csproj]: Nevermind, it seems that this project format it is not compatible yet: https://github.com/dotnet/project-system/issues/1467

**Pull request merged**